grumblr/app/assets/javascripts/application.js

```diff
@@ -10,7 +10,23 @@
 // Read Sprockets README (https://github.com/sstephenson/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require jquery
+///= require jquery
```

Comment:
I believe this change was a mistake. Did you review your diff before or after submitting the PR? That's usually a good idea.